### PR TITLE
build: add Central Kurdish translations to `nvim.desktop`

### PR DESCRIPTION
**Pull request by rojvv**

### CODE REVIEW

**Comment by dundargoc** [comment]
@clason shall I add this to the typo PR? (technically not a typo, but maybe small enough to qualify)

**Comment by clason** [comment]
I'd leave it alone; I can't verify the translation, so it's better to keep it as a separate commit.

**Comment by roj1512** [comment]
I’ll request a review from another native speaker for its verification.

File: runtime/nvim.desktop [hunk 1]
```diff
@@ -1,6 +1,7 @@
 [Desktop Entry]
 Name=Neovim
 GenericName=Text Editor
+GenericName[ckb]=دەستکاریکەری نووسین
```
Comment (Sarchia) [inline runtime/nvim.desktop:4]:
```suggestion
GenericName[ckb]=دەستکاریکەری دەق
```

File: runtime/nvim.desktop [hunk 2]
```diff
@@ -79,6 +81,7 @@ Exec=nvim %F
 Terminal=true
 Type=Application
 Keywords=Text;editor;
+Keywords[ckb]=نووسین;دەستکاریکەر;
```
Comment (Sarchia) [inline runtime/nvim.desktop:84]:
```suggestion
Keywords[ckb]=دەق;دەستکاریکەر;
```

Comment (Sarchia) [inline runtime/nvim.desktop:84]:
```suggestion
Keywords[ckb]=دەق;دەستکاریکەر;
```

File: runtime/nvim.desktop [hunk 3]
```diff
@@ -16,6 +17,7 @@ Comment[bg]=Редактиране на текстови файлове
 Comment[bn]=টেক্স্ট ফাইল এডিট করুন
 Comment[bs]=Izmijeni tekstualne datoteke
 Comment[ca]=Edita fitxers de text
+Comment[ckb]=فایلی نووسین دەستکاری بکە
```
Comment (Sarchia) [inline runtime/nvim.desktop:20]:
```suggestion
Comment[ckb]=دەستکاریی فایلی دەق بکە
```

Comment (Sarchia) [inline runtime/nvim.desktop:20]:
```suggestion
Comment[ckb]=دەستکاریی فایلی دەق بکە
```

**Comment by zeertzjq** [comment]
The commit message needs to follow conventional commits. Perhaps it should be `chore: add Central Kurdish translations to nvim.desktop`.

**Comment by roj1512** [comment]
@zeertzjq done! I have updated the PR title so they can squash and merge with that commit message.

**Comment by zeertzjq** [comment]
The commit linter CI is required for merging, which requires every commit in the PR to follow conventional commits.

**Comment by roj1512** [comment]
OK, I don’t think it makes any sense but I’ll make a force push for making it pass here!

**Comment by roj1512** [comment]
Any updates? :)

**Comment by roj1512** [comment]
Finally!

**Pull request merged**